src/background.js

```diff
@@ -372,6 +382,14 @@ async function containFacebook (options) {
   return {cancel: true};
 }
 
+function showPanel() {
+  if(isFacebookURL){
+    browser.browserAction.getPopup({});
```

Comment:
This is probably trying to call [`browserAction.openPopup()`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/browserAction/openPopup), but there's a problem: "You can only call this function from inside the handler for a user action."

So, we can't force the browserAction popup to open as soon as the user visits facebook.com.

As an alternative, in the #contain-facebook Slack channel we discussed [adding a badge](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/browserAction/setBadgeTextColor) to the `browserAction` icon to visually encourage the user to click the icon and receive the pop-up.

I'm starting a branch based on this one which starts to add this.